src/openrct2/paint/tile_element/Paint.Path.cpp

```diff
@@ -739,7 +740,7 @@ static void sub_6A3F61(
         // Redundant zoom-level check removed
 
         sub_6A4101(
-            session, tile_element, height, connectedEdges, word_F3F038, footpathEntry, footpathEntry->image | imageFlags,
+            session, tile_element, height, connectedEdges, word_F3F038, railingEntry, railingEntry->image | imageFlags,
```

Comment:
```suggestion
            session, tile_element, height, connectedEdges, word_F3F038, railingEntry, footpathEntry->image | imageFlags,
```
Signature should be changed accordingly.